packages/next/client.js

```diff
@@ -1 +1,4 @@
-module.exports = require('./dist/client/index')
+module.exports =
+  process.env.NEXT_RUNTIME === 'edge'
```

Comment:
Can we achieve this with resolve aliasing in the webpack-config instead of hardcoding it here?

Reply:
![image](https://user-images.githubusercontent.com/11064311/190508514-1f81bd71-0342-42bf-8880-e3b14b678c72.jpeg)

That was an approach I tried but this didn't work and I didn't dig much. Do you know what could be wrong in my screenshot?

Reply:
Hmm that seems correct, was it not resolving at all with that config or just still using the CJS variant?

Reply:
just resolving the CJS variant 😥

Reply:
is that a blocker for the PR @ijjk?

Reply:
Ah I see, not a blocker

Reply:
Updated the compiled although seems there are some test failures needing to be investigated still